src/history/hs.c

```diff
@@ -1123,7 +1126,7 @@ __wt_find_hs_upd(WT_SESSION_IMPL *session, WT_ITEM *key, const char *value_forma
     WT_ITEM hs_key, recno_key;
     WT_MODIFY_VECTOR modifies;
     WT_TXN *txn;
-    WT_TXN_SHARED *txn_shared;
+    WT_TXN_SHARED *txn_shared_p;
```

Comment:
Agreed. This should just be `txn_shared`

Reply:
This one was interesting. The macro WT_WITH_TXN_ISOLATION also had an internal variable `txn_shared` and that collided with the one outside the scope. 

I'll name it something that doesn't have `_p` to avoid confusion.